### PR TITLE
the JSON output dir test does not test what we think it does

### PR DESCRIPTION
**Pull request by jamestalmage**
Just documenting this. 

I suspect this is due to the fact we are running in a container instrumented by NYC

### CODE REVIEW

File: test/nyc-test.js [hunk 1]
```diff
@@ -138,37 +138,36 @@ describe('nyc', function () {
...
-              // the subprocess we ran should output reports
-              // for files in the fixtures directory.
-              Object.keys(report).should.match(/.\/test\/fixtures\//)
-            }
+      //  var proc = spawn(process.execPath, ['./test/fixtures/sigint.js'], {
+      //    cwd: process.cwd(),
+      //    env: process.env,
+      //    stdio: 'inherit'
+      //  })
+      //  var start = fs.readdirSync(nyc.tmpDirectory()).length
```
Comment (jamestalmage) [inline test/nyc-test.js]: This test works even though we never spawn another process.

**Comment by novemberborn** [comment]
#85 has a fix for this, spawning `bin/nyc.js` to run the `sigint.js` file.

**Comment by jamestalmage** [comment]
> #85 has a fix for this, spawning bin/nyc.js to run the sigint.js file.

I took the same approach in #89 :smile:

**Comment by novemberborn** [comment]
#104 fixes this.

**Pull request closed without merging**